decidim-elections/app/cells/decidim/votings/content_blocks/landing_page/description/show.erb

```diff
@@ -5,21 +5,10 @@
         <img src="<%= introductory_image.path %>" alt="">
       </div>
     <% end %>
-    <div class="columns medium-8 mediumlarge-7 content">
+    <div class="columns medium-8 mediumlarge-7">
       <h2 class="show-for-sr"><%= t("decidim.votings.content_blocks.landing_page.description.title") %></h2>
       <div><%= description_text %></div>
     </div>
-    <div class="content-height-toggler show-more">
-      <button class="button button--muted tiny">
-        <span class="button-text show-more-content">
-          <%= button_show_more_text %>
```

Comment:
Can you remove the other references too, please? 

https://github.com/decidim/decidim/blob/e2f5b9cb7edc5c4fce0b1e5d5fc75456b6cb9b0a/decidim-elections/config/locales/en.yml?plain=1#L1099 

https://github.com/decidim/decidim/blob/e2f5b9cb7edc5c4fce0b1e5d5fc75456b6cb9b0a/decidim-elections/app/cells/decidim/votings/content_blocks/landing_page/description_cell.rb#L22

https://github.com/decidim/decidim/blob/e2f5b9cb7edc5c4fce0b1e5d5fc75456b6cb9b0a/decidim-elections/app/packs/src/decidim/votings/voting-description-cell.js?plain=1#L29

https://github.com/decidim/decidim/blob/e2f5b9cb7edc5c4fce0b1e5d5fc75456b6cb9b0a/decidim-elections/app/packs/src/decidim/votings/voting-description-cell.js?plain=1#L2